src/components/files/FileAbout.tsx

```diff
@@ -9,18 +9,18 @@ type FileAboutProps = {
 }
 
 export function FileAbout(props: FileAboutProps) {
-	const {fileMetadata} = props;
+	const {id, created, name, creator, version} = props.fileMetadata;
 
 	return (
 		<Box className="infoCard">
 			<Typography className="title">About</Typography>
-			<Typography className="content">File ID: {fileMetadata["id"]}</Typography>
+			<Typography className="content">File ID: {id}</Typography>
 			{/*<Typography className="content">Type: {fileMetadata["content-type"]}</Typography>*/}
 			{/*<Typography className="content">File size: {fileMetadata["size"]}</Typography>*/}
-			<Typography className="content">Updated on: {parseDate(fileMetadata["created"])}</Typography>
-			<Typography className="content">Uploaded as: {fileMetadata["name"]}</Typography>
-			<Typography className="content">Uploaded by: {fileMetadata["creator"]}</Typography>
-			<Typography className="content">Latest Version: {fileMetadata["version"]}</Typography>
+			<Typography className="content">Updated on: {parseDate(created)}</Typography>
+			<Typography className="content">Uploaded as: {name}</Typography>
+			<Typography className="content">Uploaded by: {creator.full_name}</Typography>
```

Comment:
Hmm author full name right now is empty, do you want to replace it with email address for now?

Reply:
nvm will work on updating full name